integrations/server/test_covidcast.py

```diff
@@ -10,12 +10,13 @@
 
 # first party
 from delphi_utils import Nans
+from delphi.epidata.acquisition.covidcast.covidcast_row import CovidcastRow
 from delphi.epidata.acquisition.covidcast.test_utils import CovidcastBase
 
 # use the local instance of the Epidata API
+# TODO: should we still be using this?
```

Comment:
> `# TODO: should we still be using this?`

Yes -- all Delphi's pre-pandemic users probably still use the old old delphi-epidata client, which means they probably still hit the api.php endpoint.